diff --git a/node_modules/@remix-run/dev/dist/vite/plugin.js b/node_modules/@remix-run/dev/dist/vite/plugin.js
index c742d24..1c614fe 100644
--- a/node_modules/@remix-run/dev/dist/vite/plugin.js
+++ b/node_modules/@remix-run/dev/dist/vite/plugin.js
@@ -421,7 +421,7 @@ const remixVitePlugin = (options = {}) => {
           // Pre-bundle React dependencies to avoid React duplicates,
           // even if React dependencies are not direct dependencies.
           // https://react.dev/warnings/invalid-hook-call-warning#duplicate-react
-          "react", "react/jsx-runtime", "react/jsx-dev-runtime", "react-dom/client",
+          "react", "react-dom/client",
           // Pre-bundle Remix dependencies to avoid Remix router duplicates.
           // Our remix-remix-react-proxy plugin does not process default client and
           // server entry files since those come from within `node_modules`.
@@ -437,7 +437,8 @@ const remixVitePlugin = (options = {}) => {
         },
         esbuild: {
           jsx: "automatic",
-          jsxDev: viteCommand !== "build"
+          jsxDev: viteCommand !== "build",
+          jsxImportSource: "@emotion/react"
         },
         resolve: {
           dedupe: [
