diff --git a/node_modules/@remix-run/dev/dist/vite/plugin.js b/node_modules/@remix-run/dev/dist/vite/plugin.js
index 53caa39..352d388 100644
--- a/node_modules/@remix-run/dev/dist/vite/plugin.js
+++ b/node_modules/@remix-run/dev/dist/vite/plugin.js
@@ -418,11 +418,14 @@ const remixVitePlugin = (options = {}) => {
           hmrPartialAccept: true
         },
         optimizeDeps: {
+          esbuildOptions: {
+            jsx: "automatic"
+          },
           include: [
           // Pre-bundle React dependencies to avoid React duplicates,
           // even if React dependencies are not direct dependencies.
           // https://react.dev/warnings/invalid-hook-call-warning#duplicate-react
-          "react", "react/jsx-runtime", "react/jsx-dev-runtime", "react-dom/client",
+          "react", "@emotion/react/jsx-runtime", "@emotion/react/jsx-dev-runtime", "react-dom/client",
           // Pre-bundle Remix dependencies to avoid Remix router duplicates.
           // Our remix-remix-react-proxy plugin does not process default client and
           // server entry files since those come from within `node_modules`.
@@ -438,8 +441,10 @@ const remixVitePlugin = (options = {}) => {
         },
         esbuild: {
           jsx: "automatic",
-          jsxDev: viteCommand !== "build"
+          jsxImportSource: "@emotion/react",
+          jsxDev: viteCommand !== "build",
         },
+        optimizeDeps: { esbuildOptions: { jsx: "automatic" } },
         resolve: {
           dedupe: [
           // https://react.dev/warnings/invalid-hook-call-warning#duplicate-react
@@ -836,7 +841,7 @@ const remixVitePlugin = (options = {}) => {
       if (id.includes("/node_modules/")) return;
       let [filepath] = id.split("?");
       if (!/.[tj]sx?$/.test(filepath)) return;
-      let devRuntime = "react/jsx-dev-runtime";
+      let devRuntime = "@emotion/react/jsx-dev-runtime";
       let ssr = (options === null || options === void 0 ? void 0 : options.ssr) === true;
       let isJSX = filepath.endsWith("x");
       let useFastRefresh = !ssr && (isJSX || code.includes(devRuntime));
@@ -848,7 +853,7 @@ const remixVitePlugin = (options = {}) => {
           sourceType: "module",
           allowAwaitOutsideFunction: true
         },
-        plugins: [[require("react-refresh/babel"), {
+        plugins: ['babel-plugin-macros', '@emotion/babel-plugin', [require("react-refresh/babel"), {
           skipEnvCheck: true
         }]],
         sourceMaps: true
